_pages/docs/matching-options.md

```diff
@@ -0,0 +1,159 @@
+used to find the code.
+
+### Scoring
+
+Matches are based on how many properties match, what algorithms are used to
+determine matches, and the weights placed on the properties. See the How Scoring
+Works section for more information about how match scores are calculated.
+
+#### `add` elements
+
+This section really drives the matching process. For each `add` element, the
+matcher will add this element or property to the query used to find other
+documents that match the initial document. The `property-name` attribute must
+match the `name` attribute of a `property` element defined under
+`property-defs`. The `weight` attribute is the number of points awarded to
+a potential match if the values match.
+
+#### `expand` elements
+
+To apply a different matching strategy, add an `extend` element. The
+`property-name` element must match the `name` attribute of a `property` element
+defined under `property-defs`. The `algorithm-ref` attribute must match the
+`name` attribute of an `algorithm` element under `algorithms`. Sub-elements
+under the `extend` element depend on the implementation of the function.
+
+#### `reduce` elements
+
+In some cases, a combination of matching properties may suggest a match when
+there shouldn't be one. For instance, two Person records have the same family
+name, same street address, city, and zip code. That might be enough points to
+trigger a match, but if the given names are different, it's probably relatives
+living together. The `reduce` element gives a way to back off the scores in
+such cases. The `algorithm-ref` attribute must match the `name` attribute of an
+`algorithm` element under `algorithms`. The `weight` attribute will be
+subtracted from the score if the algorithm matches.
+
+### Thresholds
+
+Smart-mastering-core can be configured to take different actions based on the
+match scores. Each `threshold` element has an `above` attribute, which is the
+minimum absolute score a match must have against the initial document in order
+to reach that threshold. A `threshold` element may also have a `label` attribute
+and an `action` attribute. If the `action` attribute is "merge", then the
+initial document and any document that reaches this threshold will be
+automatically merged. For `action="notify"`, a notification will be recorded
+for a human reviewer.
+
+### Tuning
+
+The `max-scan` element limits the number of potential matches that get
+processed. In the example configuration with `<max-scan>200</max-scan>`,
+only the top 200 scoring potential matches will be merged or have notifications
+recorded.
+
+## How Scoring Works
+
+High scores are relative to the configuration, rather than measured on an
+absolute scale. The maximum possible score is the sum of the weights of all of
+the `scoring/add/@weight` attributes. In the example configuration above, the
+maximum possible score is 50+8+6+5+3+1+3+6+8-4=86. Thresholds must be adjusted
```

Comment:
I didn't think of this for the other page... but maybe we should include the excerpt from above to make it super obvious...

```
<add property-name="ssn" weight="50"/>
<add property-name="last-name" weight="8"/>
<add property-name="first-name" weight="6"/>
<add property-name="addr1" weight="5"/>
<add property-name="city" weight="3"/>
<add property-name="state" weight="1"/>
<add property-name="zip" weight="3"/>
<expand property-name="first-name" algorithm-ref="thesaurus" weight="6">
<expand property-name="last-name" algorithm-ref="dbl-metaphone" weight="8">
<reduce algorithm-ref="std-reduce" weight="4">
```
50+8+6+5+3+1+3+6+8-4=86